boskos/Makefile

```diff
@@ -73,10 +74,16 @@ metrics-deployment:
 service:
 	kubectl apply -f service.yaml
 
-update-config: get-cluster-credentials
+activate-prow:
+	if [ -a $(SERVICE_ACCOUNT) ] ; \
+	then \
+		gcloud auth activate-service-account --key-file=$(SERVICE_ACCOUNT) ;\
+	fi;
+
+update-config: activate-prow get-cluster-credentials
```

Comment:
Any way we can make the activate-prow part optional? Or is the assumption that we never run this manually now?

Reply:
I tried locally, run manually is still fine, it will skip that entire if check